string group_name
moveit_msgs/CollisionObject place_object
